challenge_2/python/CindyB/src/singleNumber.py

```diff
@@ -0,0 +1,8 @@
+def findSingleNumber(myArray):
+    for x in myArray:
+        count = sum(map(lambda s: s == x,myArray))
```

Comment:
You said in your README that you wanted to solve this in O(n). This solution is definitely not O(n). In fact, it's O(n^2).

You can think of the `sum` and `map` operations as for loops because in order for python to map an array and sum the contents of the array, it has to go through the entire array. There are some tricks that python can do but in the end, you have a nested for loop here.